zcash_keys/src/address.rs

```diff
@@ -234,6 +234,17 @@ impl UnifiedAddress {
         );
         result.collect()
     }
+
+    /// Returns the set of receivers in the unified address, excluding unknown receiver types.
+    pub fn receivers(&self) -> Vec<Receiver> {
```

Comment:
I dislike that `UnifiedAddress::receivers` is not symmetric to `UnifiedAddress::receiver_types`. Either return unknown receivers from this method, or rename it to be explicit that it returns the subset of known receivers.